### ISSUE TITLE
Failed to parse source map "node_modules/react-signalr/index.js.map"

### ISSUE THREAD

**Issue by dattran1232003** [hosseinmd/react-signalr#36]
Hi, I got a warning when run a ReactJS project.

```
Compiled with warnings.

Failed to parse source map from '/Volumes/Data/works/kai-dev/ClientApp/node_modules/react-signalr/index.js.map' file: Error: ENOENT: no such file or directory, open '/Volumes/Data/works/kai-dev/ClientApp/node_modules/react-signalr/index.js.map'

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

WARNING in ./node_modules/react-signalr/signalr.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Volumes/Data/works/kai-dev/ClientApp/node_modules/react-signalr/index.js.map' file: Error: ENOENT: no such file or directory, open '/Volumes/Data/works/kai-dev/ClientApp/node_modules/react-signalr/index.js.map'

webpack compiled with 1 warning
Files successfully emitted, waiting for typecheck results...
Issues checking in progress...
No issues found.
```

package versions:
```
{
    "@microsoft/signalr": "^8.0.0",
    "react-signalr": "^0.2.18",
}
```

yarn.lock:
```
"@microsoft/signalr@^7.0.10":
  version "7.0.14"
  resolved "https://registry.yarnpkg.com/@microsoft/signalr/-/signalr-7.0.14.tgz#97551d68f09bb0ae063ffec738f7398f29d297ff"
  integrity sha512-dnS7gSJF5LxByZwJaj82+F1K755ya7ttPT+JnSeCBef3sL8p8FBkHePXphK8NSuOquIb7vsphXWa28A+L2SPpw==
  dependencies:
    abort-controller "^3.0.0"
    eventsource "^2.0.2"
    fetch-cookie "^2.0.3"
    node-fetch "^2.6.7"
    ws "^7.4.5"

"@microsoft/signalr@^8.0.0":
  version "8.0.0"
  resolved "https://registry.yarnpkg.com/@microsoft/signalr/-/signalr-8.0.0.tgz#cb1412e88e0527f40da9178fefc27a65c3ddeab0"
  integrity sha512-K/wS/VmzRWePCGqGh8MU8OWbS1Zvu7DG7LSJS62fBB8rJUXwwj4axQtqrAAwKGUZHQF6CuteuQR9xMsVpM2JNA==
  dependencies:
    abort-controller "^3.0.0"
    eventsource "^2.0.2"
    fetch-cookie "^2.0.3"
    node-fetch "^2.6.7"
    ws "^7.4.5"
    
react-signalr@^0.2.18:
  version "0.2.18"
  resolved "https://registry.yarnpkg.com/react-signalr/-/react-signalr-0.2.18.tgz#b5d3d4821a1c644d2b6b20b678ce1b602d089d7d"
  integrity sha512-iu3mMRUqc2ruLkab7e+yEj6dgYYwRhENNYfz2/AXc3wBVSVU8nlNw5IXOpGdrIII4XtRwVsMtResL8pU3ph3EA==
  dependencies:
    "@microsoft/signalr" "^7.0.10"
    hermes-channel "^2.1.2"
    js-cookie "^2.2.1"
    socket.io "^4.2.0"
    socket.io-client "^4.2.0"
    uuid "^8.3.2"
```

Thank you for creating such a great package!

**Comment by aukgit** [comment]
@hosseinmd it is a valid issue, please fix it.

**Comment by hosseinmd** [comment]
Fixed: v0.2.21
